src/fty_common_utf8.cc

```diff
@@ -413,6 +413,23 @@ s_jsonify_translation_string (const char *key, va_list args)
     std::string json_str (json_attempt);
     free (json_attempt);
 
+    // drop quotes enclosing inserted variables which are already in JSON format
+    // - one from the previous call, second from this one
+    // (backslashes were escaped, so we need to drop them as well):
+    size_t insert_start = json_str.find ("\"\\\"{");
```

Comment:
This should be called in a loop for all occurences, as we might put multiple variables containing JSON into the output

Reply:
Updated - if you meant something else, please provide a test case :-)